src/components/ModalManager/ModalManager.js

```diff
@@ -43,14 +43,16 @@ const ModalView = () => (
   <ModalConsumer>
     {({ modalComponent: ModalComponent, modalComponentProps, hideModal }) => (
       <Transition
-        blocking={!!ModalComponent}
         native
+        items={Boolean(ModalComponent)}
+        blocking={Boolean(ModalComponent)}
```

Comment:
Was this working? I would get `undefined` for `blocking`. [This](https://github.com/aragon/aragon-ui/pull/240/files#diff-7f106da13e09b507dfe3404d2184836cR54) is how I fixed it.

Reply:
In the [documentation](http://react-spring.surge.sh/transition) it reads as important to:
>You should always rely on the item that you receive instead of reading from scope, since you could be rendering an item that has already been deleted, but is retained inside Transition until its fade-out is complete.

That's why I re-scoped the Component.

Reply:
Thanks @AquiGorka! In the end I passed `ModalComponent` as the item, and `blocking` is part of the transition parameters: https://github.com/aragon/aragon/pull/411/commits/f66fc86b118df8725702091004bb9fbbe5dde483

Reply:
Nice! I like your code, will implement in my PR.